JamulusWin.ps1

```diff
@@ -0,0 +1,384 @@
+#==================================================================================================
+# Visual Studio Tool Script by pgScorpio
+#==================================================================================================
+Add-Type -AssemblyName 'PresentationFramework'
+#==================================================================================================
```

Comment:
Could you write a script which gets the files from qmake and modifies them? 
Why should we include vs files directly in the repo? They should be EASILY generated by qmake - or a script using qmake. If there are manual steps, they should be documented in COMPILING.md

Reply:
@ann0see 

> Could you write a script which gets the files from qmake and modifies them? Why should we include vs files directly in the repo? They should be EASILY generated by qmake - or a script using qmake. If there are manual steps, they should be documented in COMPILING.md

Of coarse anything is possible, but it would not be easy. Don't forget these files are in my repo because I use them for development and I want them under version control too. They don't need to be included when merging to master.